src/app/assets/icons/PXBLogo.tsx

```diff
@@ -38,7 +38,7 @@ export const PXBLogo: React.FC<PXBLogoProps> = ({ tagline, color }) => {
         <div style={{ textAlign: 'center', position: 'relative' }}>
             <PXBlueSVG className={'rotateMe'} size={82} color={color || theme.palette.primary.contrastText} />
             <Typography variant={'h3'} align={'center'} style={{ fontWeight: 300, marginBottom: theme.spacing() }}>
-                Power Xpert <b>Blue</b>
+                Brightlayer UI
```

Comment:
This one too. The original version spelled the full name out + bolded the "blue", so this one shall do the same